README.md

```diff
@@ -139,19 +139,20 @@ Assuming the name of the created Sandbox is named `foo`, the following resources
 
 ### Namespace (sandbox-foo)
 
-### ClusterRole (sandbox-foo-deleter)
+### ClusterRole (sandbox-foo-admins)
```

Comment:
nit: It looks like we flipped all of the Role/RoleBinding pluralities. The backing code looks correct.

Reply:
Nope, good for our docs to match our code.  Someday we'll have automated test for docs :-)